python/tvm/ir/op.py

```diff
@@ -17,9 +17,21 @@
 # pylint: disable=invalid-name
 """Primitive operators in the TVM IR."""
 import tvm._ffi
+from tvm._ffi.base import _FFI_MODE
 from .expr import RelayExpr
 from . import _ffi_api
 
+try:
+    # pylint: disable=wrong-import-position,unused-import
+    if _FFI_MODE == "ctypes":
+        raise ImportError()
+    from tvm._ffi._cy3.core import convert_to_tvm_func, _get_global_func, PackedFuncBase
+except (RuntimeError, ImportError) as error:
+    # pylint: disable=wrong-import-position,unused-import
+    if _FFI_MODE == "cython":
+        raise error
+    from tvm._ffi._ctypes.packed_func import convert_to_tvm_func, _get_global_func, PackedFuncBase
```

Comment:
😆 i suppose these lines become useless given you have addressed the comments i left before?

Reply:
Removed, thanks!